pywebostv/controls.py

```diff
@@ -266,9 +251,9 @@ class SystemControl(WebOSControlBase):
             "uri": "ssap://system.notifications/createToast",
             "args": [str],
             "payload": {
-                "iconData": arguments(1, postprocess=lambda url: get_icon_data(url), default=None) ,
-                "iconExtension": "png",
                 "message": arguments(0),
+                "iconData": arguments("icon_bytes", default=None) ,
```

Comment:
Thanks a lot for changes, really appreciate it.

One tiny question: Doesn't LG want the bytes to be encoded in Base64 and presented as a UTF-8 string (according to your previous commit)? If so, can we use `postprocess=` param within this argument?

Reply:
Done.